examples/simple_sampler/testing_library/testing_library.h

```diff
@@ -0,0 +1,7 @@
+#pragma once
```

Comment:
Rename to test_tracer.h? testing_library is a little too generic.

Reply:
Updated